### PR TITLE
Add warning for scale merged.

### PR DESCRIPTION
**Pull request by kanitw**

### CODE REVIEW

File: examples/vg-specs/layer_text_heatmap.vg.json [hunk 1]
```diff
@@ -225,6 +225,9 @@
                         "angle": {
                             "value": 270
                         },
+                        "align": {
+                            "value": "right"
+                        },
```
Comment (arvind) [inline examples/vg-specs/layer_text_heatmap.vg.json]: I'm a little confused about where this addition came from as there's no associated edit to a `vl.json` spec?

Comment (kanitw) [inline examples/vg-specs/layer_text_heatmap.vg.json]: It's artifact from a former commit that got rebase merged.

File: src/compile/scale/properties.ts [hunk 2]
```diff
@@ -102,6 +102,7 @@ export function parseNonUnitScaleProperty(model: Model, property: keyof (Scale |
         // TODO: precedence rule
         valueWithExplicit = mergeValuesWithExplicit(
           valueWithExplicit, childValueWithExplicit,
+          property,
```
Comment (domoritz) [inline src/compile/scale/properties.ts]: How come you are adding a new parameter here?

Comment (kanitw) [inline src/compile/scale/properties.ts]: Well, you need to know what's the property to make the warning message include the property name.

**Pull request merged**